docroot/themes/contrib/civictheme/civictheme_library/components/02-molecules/pagination/pagination.stories.js

```diff
@@ -66,7 +72,7 @@ export const Pagination = (knobTab) => {
     } : null,
     current: number(
       'Current page',
-      Math.floor(pageCount / 2),
+      1,
```

Comment:
Ensures we always have a valid current number - previously - what would happen if we had 1 page...

Reply:
`Math.max(1, Math.floor(pageCount / 2)),`